packages/boot/tools/utils.ts

```diff
@@ -0,0 +1,58 @@
+import { encodeAbiParameters, toBytes, type Hex } from 'viem';
+import { encodeBase64 } from '@endo/base64';
+
+export type CallResult = {
+  success: boolean;
+  result: Hex;
+};
+
+export type AgoricResponse = {
+  isContractCallResult: boolean;
+  data: CallResult[];
+};
+
+/**
+ * Encodes a payload according to Agoric's response structure
+ * @param {object} response - The response object matching AgoricResponse struct
+ * @param {boolean} response.isContractCallResult
+ * @param {Array<CallResult>} response.data
```

Comment:
I wonder why this is repeated. why not...

```suggestion
 * @param {AgoricResponse} response
```